UnityProject/Assets/Scripts/Clothing/Eyewear/NightVisionGoggles.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using CameraEffects;
+
+public class NightVisionGoggles : MonoBehaviour, IServerInventoryMove
+{
+	[SerializeField, Tooltip("How far the player will be able to see in the dark while he has the goggles on.")]
+	private Vector3 nightVisionVisibility;
+
+	[SerializeField, Tooltip("The default minimal visibility size.")]
+	private Vector3 defaultVisionVisibility;
+
+
+	public void OnInventoryMoveServer(InventoryMove info)
+		{
+			RegisterPlayer registerPlayer = info.ToRootPlayer;
+
+			if (info.ToSlot != null && info.ToSlot?.NamedSlot != null)
+			{
+
+				if (registerPlayer != null && info.ToSlot.NamedSlot == NamedSlot.eyes)
+				{
+					TargetOnWearing();
```

Comment:
you'll need to add the parameter, here. not sure if info.conn is a thing

Reply:
still need to solve this issue first https://github.com/unitystation/unitystation/pull/6112#issuecomment-785094553

Reply:
change the class from a monobehaviour to NetworkBehaviour

Reply:
still nothing

Reply:
using Mirror

Reply:
![image](https://user-images.githubusercontent.com/34368774/109014097-7ee81400-76bc-11eb-8b6f-646f538a58d9.png)